launcher/container_runner.go

```diff
@@ -99,22 +99,24 @@ func NewRunner(ctx context.Context, cdClient *containerd.Client, token oauth2.To
 		container.Delete(ctx, containerd.WithSnapshotCleanup)
 	}
 
-	logger.Printf("Operator Input Image Ref   : %v\n", image.Name())
-	logger.Printf("Image Digest               : %v\n", image.Target().Digest)
-	logger.Printf("Operator Override Env Vars : %v\n", envs)
-	logger.Printf("Operator Override Cmd      : %v\n", launchSpec.Cmd)
+	logger.Info("Preparing Container Runner",
+		"operator_input_image_ref", image.Name(),
+		"image_digest", image.Target().Digest,
+		"operator_override_env_vars", envs,
+		"operator_override_cmd", launchSpec.Cmd,
+	)
 
 	imageConfig, err := getImageConfig(ctx, image)
 	if err != nil {
 		return nil, err
 	}
 
-	logger.Printf("Exposed Ports:             : %v\n", imageConfig.ExposedPorts)
+	logger.Info(fmt.Sprintf("Exposed Ports:             : %v\n", imageConfig.ExposedPorts))
```

Comment:
nit: can we pretty print programmatically vs with hardcoded spaces? maybe a function that calculates the longest string and pads every other one?

Reply:
I realize this isn't the point of your PR so it's okay if we don't do this.

Reply:
In the interest of getting this PR submitted soon (syncing to main is a huge hassle given how many files this changes) I'll add formatting changes - including a better attestation token log - in a different PR.